libs/sdk-ui-kit/src/Dialog/EmbedInsightDialog/EmbedInsightDialogBase/components/PrepareEnvMessage.tsx

```diff
@@ -0,0 +1,31 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import { FormattedMessage } from "react-intl";
+import { Message } from "../../../../Messages";
+
+export interface IPrepareEnvMessageProps {
+    isTiger: boolean;
+}
+
+const bearLink = "https://sdk.gooddata.com/gooddata-ui/docs/platform_integration.html";
+const tigerLink = "https://sdk.gooddata.com/gooddata-ui/docs/cloudnative_integration.html";
+
+export const PrepareEnvMessage: React.VFC<IPrepareEnvMessageProps> = (props) => {
+    const { isTiger } = props;
+
+    const link = isTiger ? tigerLink : bearLink;
```

Comment:
I think we don't have condition like this anywhere in the SDK and for a good reason.
Is there some other way to handle / name this condition?

Reply:
Moved upper level link will be provided